xds/src/main/java/com/linecorp/centraldogma/xds/k8s/v1/XdsKubernetesEndpointFetchingService.java

```diff
@@ -236,32 +232,33 @@ private static class KubernetesEndpointsUpdater {
             this.groupName = groupName;
             this.aggregator = aggregator;
 
+            CompletableFutures.successfulAsList(kubernetesEndpointGroupFutures, t -> null)
+                              .thenAccept(endpointGroups -> {
+                                  final List<CompletableFuture<List<com.linecorp.armeria.client.Endpoint>>>
+                                          whenReadys = endpointGroups.stream()
+                                                                     .filter(Objects::nonNull)
+                                                                     .map(KubernetesEndpointGroup::whenReady)
+                                                                     .collect(toImmutableList());
+                                  CompletableFutures.successfulAsList(whenReadys, t -> null)
```

Comment:
Question) I may have missed this and it may be irrelevant to this PR, but will the failed (`whenReadys` || `kubernetesEndpointGroupFutures`) be logged anywhere?

Reply:
Good point. Updated. 👍